lib/bundles/inspec-compliance/api.rb

```diff
@@ -83,8 +83,13 @@ def self.version(config)
     # verifies that a profile
     def self.exist?(config, profile)
       _msg, profiles = Compliance::API.profiles(config)
+      owner, id, ver = profile_split(profile)
       if !profiles.empty?
-        index = profiles.index { |p| "#{p['owner_id']}/#{p['name']}" == profile }
+        index = profiles.index { |p|
```

Comment:
We never really use the index except for checking to see if we have a match. Could we refactor this to use `#any?` instead while we're in here?

```ruby
profiles.any? do |p|
  # criteria
end
```

... that should return true if there was a match, false if not.